README.md

```diff
@@ -128,10 +128,10 @@ The RAPIDS versions for things like container images and install instructions ar
 ```python
 versions = {
     "stable": {
-        "rapids_container": "nvcr.io/nvidia/rapidsai/rapidsai-core:23.02-cuda11.8-runtime-ubuntu22.04-py3.10",
+        "rapids_container": "nvcr.io/nvidia/rapidsai/base:24.04-cuda11.8-py3.10",
     },
     "nightly": {
-        "rapids_container": "rapidsai/rapidsai-core-nightly:23.04-cuda11.8-runtime-ubuntu22.04-py3.10",
+        "rapids_container": "rapidsai/base:24.06a-cuda11.8-py3.10",
```

Comment:
`rapids/rapidsai-core` and `rapidsai/rapidsai-core-nightly` stopped receiving updates about a year ago:

* https://docs.rapids.ai/notices/rsn0030/
* https://github.com/rapidsai/docker/issues/539

This proposes updating this part of the README to eliminate another reference to the old no-longer-supported images.

*(there are more of these in the repo, but that can be a separate PR)*